sprin10/script.js

```diff
@@ -0,0 +1,32 @@
+function findSquare(num) {
+  return num * num;
+}
+
+// Example usage
+console.log(findSquare(6));  // Output: 36
+console.log(findSquare(0));  // Output: 0
+console.log(findSquare(-12));  // Output: 144
+
+function getRandom(start, end) {
+  return Math.floor(Math.random() * (end - start)) + start;
+}
+
+// Example usage
+console.log(getRandom(3, 5));  // Output: Random number between 3 and 5 (e.g., 3 or 4)
+
+function letterCount(sir, litera) {
+  const regex = new RegExp(litera, 'gi'); // 'gi' for case-insensitive search
```

Comment:
Oare ai inteles cum functioneaza regExp? 

Recomand sa facem fara regexp, dar cu for-loop, ca sa ne obisnuim cu a scrie cod "for"